api/src/main/java/marquez/cli/SeedCommand.java

```diff
@@ -0,0 +1,1419 @@
+        log.info("Created run for job '{}': {}", jobName, run);
+
+        // Start run
+        final Run running = client.markRunAsRunning(run.getId(), runStartedAt);
+        log.info("Marked run for job '{}' as 'RUNNING': {}", jobName, running);
+
+        // Complete or fail run
+        if (activeRunMeta.isMarkFailed()) {
+          final Run failed = client.markRunAsFailed(run.getId(), runEndedAt);
+          log.info("Marked run for job '{}' as 'FAILED': {}", jobName, failed);
+        } else if (activeRunMeta.isMarkRunning()) {
+          for (final DatasetId output : jobMeta.getOutputs()) {
+            final DatasetMeta datasetMeta = DATASET_META.get(output.getName());
+            final DbTableMeta.DbTableMetaBuilder dbTableMetaWithChange =
+                DbTableMeta.builder()
+                    .physicalName(datasetMeta.getPhysicalName())
+                    .sourceName(datasetMeta.getSourceName())
+                    .description(datasetMeta.getDescription().orElse(null))
+                    .runId(run.getId());
+
+            if (activeRunMeta.hasSchemaChange()) {
+              activeRunMeta
+                  .schemaChangeFor(output.getName())
+                  .ifPresent(
+                      schemaChange ->
+                          dbTableMetaWithChange.fields(
+                              fieldsWithChange(datasetMeta.getFields(), schemaChange)));
+            } else {
+              dbTableMetaWithChange.fields(datasetMeta.getFields());
+            }
+            final Dataset modifiedDataset =
+                client.createDataset(
+                    NAMESPACE_NAME, output.getName(), dbTableMetaWithChange.build());
+            log.info(
+                "Dataset '{}' modified by job '{}' on run '{}': {}",
+                output.getName(),
+                jobName,
+                run.getId(),
+                modifiedDataset);
+          }
+
+          final Run completed = client.markRunAsCompleted(run.getId(), runEndedAt);
+          log.info("Marked run for job '{}' as 'COMPLETED': {}", jobName, completed);
+        }
+      }
+    }
+  }
+
+  int secondsToAdd() {
+    return new Random().nextInt((RUN_TIME_IN_SEC_MAX - RUN_TIME_IN_SEC_MIN) + 1)
+        + RUN_TIME_IN_SEC_MIN;
+  }
+
+  List<Field> fieldsWithChange(List<Field> fields, ActiveRunMeta.SchemaChange change) {
+    ImmutableList.Builder<Field> fieldsWithChange = ImmutableList.builder();
+    for (final Field field : fields) {
+      fieldsWithChange.add(
+          (change.getFieldName().equals(field.getName()))
+              ? Field.builder()
+                  .name(field.getName())
+                  .type(change.getToType())
+                  .tags(field.getTags())
+                  .description(field.getDescription().orElse(null))
+                  .build()
+              : field);
+    }
+    return fieldsWithChange.build();
+  }
+
+  static final ImmutableMap<String, DbTableMeta> DATASET_META =
```

Comment:
Seems like something that would ultimately be better as a yaml or json file. Not a blocking comment, but just a thought.

Reply:
We originally had the metadata in a `json` file, but became increasingly hard to manage and switch to using the java client instead vs `json` + curl cmds.

Reply:
Yeah, I can see why the logic makes sense to have in the Java command- it's mostly the data I was thinking to pull into a config file.

Reply:
The data can get a bit complex. For example, defining how many runs a job should have, for what run a code change was introduced, associating a run with a transitions etc. I think a yaml file would make a lot of sense when we switch to using OpenLineage events, it was just easier to define the logic in code.